pkg/device/probe.go

```diff
@@ -98,6 +98,11 @@ func (d Device) FSType() string {
 	return d.UDevData["ID_FS_TYPE"]
 }
 
+// FSUUID returns the filesystem UUID.
```

Comment:
fix probe like https://github.com/balamurugana/directpv/blob/crd-based-init-devel/pkg/device/probe.go

Reply:
can i do this change in a separate PR